src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer.Protocol/DelegatedTypes.cs

```diff
@@ -12,39 +12,37 @@
 namespace Microsoft.AspNetCore.Razor.LanguageServer.Protocol;
 
 internal record DelegatedSpellCheckParams(
-    VersionedTextDocumentIdentifier HostDocument);
+    RazorVersionedTextDocumentIdentifier Identifier);
```

Comment:
Just a note that we need to follow this up with an equivalent change in the C# extension. Unavoidable though